cni-plugin/integration/run.sh

```diff
@@ -60,31 +94,46 @@ k run linkerd-proxy \
     --rm \
     -- \
     /usr/lib/linkerd/linkerd2-network-validator --log-format plain \
-      --log-level debug --connect-addr 1.1.1.1:20001 \
-      --listen-addr 0.0.0.0:4140 --timeout 10s
+    --log-level debug --connect-addr 1.1.1.1:20001 \
+    --listen-addr 0.0.0.0:4140 --timeout 5m
```

Comment:
Were you seeing issues that required raising this timeout? I tried to stick with what we're deploying in customer's clusters.

Reply:
You know what, you were completely right. I thought this timeout was passed to `kubectl` not the validator 🤦🏻